content/articles/How-to-start-up-with-Socket-Programming-with-Java/index.md

```diff
@@ -2,22 +2,22 @@
 This is an article about Sockets. It will give you a glimpse of how clients and servers communicate over the internet.
 
 Table of content:
-- [What is a socket](#what-is-a-socket)
-- [Programming on Server-side](#programming-on-server-side)
-- [](#)
-- [](#)
-- [](#)
+- [socket definition](#socket-definition)
```

Comment:
Two of the links in the table of contents don't seem to work. Those were the server-side programming link, and the one on running the application. Make sure you check that it works using a markdown viewer. I'm guessing VS code has an extension or something to view how markdown would look when complied. Also, I know WebStorm IDE has a built-in markdown viewer if you use that IDE. As well, you can just look at the complied markdown by looking at this commit, clicking the three dots on the upper right corner (`...`) and clicking `view file`.